recipe/activate.sh

```diff
@@ -6,20 +6,21 @@ ERROR=false
 [[ "@cross_target_platform@" == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
 [[ "@cross_target_platform@" == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
 
+CUDA_FLAGS=""
 if [ "${CONDA_BUILD:-0}" = "1" ]; then
-    CUDA_INCLUDE_DIR="${PREFIX}/${targetsDir}/include"
+    CUDA_FLAGS="${CUDA_FLAGS} -I${PREFIX}/${targetsDir}/include"
+    CUDA_FLAGS="${CUDA_FLAGS} -I${BUILD_PREFIX}/${targetsDir}/include"
+    CUDA_FLAGS="${CUDA_FLAGS} -L${BUILD_PREFIX}/lib/stubs"
     # Needed to fix cross compilation
     export CMAKE_ARGS="${CMAKE_ARGS} -DCMAKE_FIND_ROOT_PATH=$PREFIX;$BUILD_PREFIX/$HOST/sysroot;$BUILD_PREFIX/${targetsDir}"
 else
-    CUDA_INCLUDE_DIR="${CONDA_PREFIX}/${targetsDir}/include"
+    CUDA_FLAGS="${CUDA_FLAGS} -I${CONDA_PREFIX}/${targetsDir}/include"
+    CUDA_FLAGS="${CUDA_FLAGS} -L${CONDA_PREFIX}/lib/stubs"
```

Comment:
I am not sure for general users (outside of the conda-build environment) this line is appropriate, is this nvcc's default behavior? If not, perhaps we should leave it out for now? I'd proceed with great caution.

Reply:
The default profile does link stubs. The profile patch as easy reference: https://github.com/conda-forge/cuda-nvcc-impl-feedstock/blob/main/recipe/nvcc.profile.patch

Reply:
Sorry, Alex (and John). This is good to know, thanks, but I realized I asked the wrong question. The real question is: Does CTK tell gcc/g++ to link to stubs, when only host compilation happens? I don't think so. We should remove this line. It doesn't matter what nvcc does, `nvcc.profile` already takes care of that as you pointed out. But here the changes are for host-only compilation.

Reply:
But if we consider to make conda users' life easier, it would justify the addition here. (Sorry I seem to be going back and forth, just wasn't sure what `cuda-nvcc` really means to conda-forge as it's new).

Reply:
Ah, no, I would not expect that as default behavior with our other installers. It is probably worth at least trying to add this here if it is helpful to users.

Reply:
> Does CTK tell gcc/g++ to link to stubs, when only host compilation happens?

It does not.

I agree I am hesitant to have the `stubs/` directory on the link line implicitly

Reply:
Should we expect that gcc/g++ builds would work without manipulating `LDFLAGS` to include this `stubs/` path if the system has a CUDA driver installed? (i.e. not in CI but a real user machine with CUDA installed)